wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/graph/LinkedGraphMaker.scala

```diff
@@ -50,12 +50,38 @@ object LinkedGraphMaker {
     graph.topologicalSort match {
       case Left(_) =>
         graph.findCycle match {
-          case Some(cycle) =>
-            val edgeStrings = cycle.value.edges map { case graph.EdgeT(from, to) => s""""${nodeName(from)}" -> "${nodeName(to)}"""" }
+          case Some(cycle) if cycle.size > 0 =>
```

Comment:
```suggestion
          case Some(cycle) if cycle.nonEmpty =>
```

Reply:
Does `Some(cycle)` need to be rewritten as `Option(cycle)`?

Reply:
oh no, just the constructions ala `Option.apply`. I believe the pattern matches have to be `Some(...)` and `None`.